charts/seed-bootstrap/charts/nginx-ingress/templates/controller-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: nginx-ingress
+    component: "controller"
+    heritage: Tiller
+    release: addons
+  name: nginx-ingress-controller
+  namespace: garden
+data:
+  server-name-hash-bucket-size: "256"
+  use-proxy-protocol: "false"
```

Comment:
I guess this needs to be configurable somehow because this setting won't work on all infrastructures. Maybe we should introduce a provider config in the `Seed` resource where such things can be specified 👀